nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/property-table/editors/nf-editor/nf-editor.component.ts

```diff
@@ -181,7 +174,7 @@ export class NfEditor implements OnDestroy {
         };
     }
 
-    resized(rect: DOMRect): void {
```

Comment:
In the template you will also need to remove the variable passed into this method.
```
[INFO] ERROR in src/app/ui/common/property-table/editors/nf-editor/nf-editor.component.html:18:96 - error TS2554: Expected 0 arguments, but got 1.
[INFO] 
[INFO] 18 <div class="property-editor p-4" [style.width.px]="width" cdkDrag resizable (resized)="resized($event)">
```

Reply:
I saw the unused parameter and just removed it before pushing the PR. Thanks for catching this!